osf/utils/fields.py

```diff
@@ -21,6 +21,17 @@ def get_prep_value(self, value):
             value = value.lower()
         return value
 
+class LowercaseEmailField(models.EmailField):
+    # TODO: This is technically not compliant with RFC 822, which requires
```

Comment:
Fair.

This text exists in at least one other place in the codebase, should it be changed there too?